Bottom: a06b284f142de4fe49fcec3f72f897fe767c44ac
Top:    62b39660db73dc8788d632994d7497faef7b4dca
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-28 22:35:27 +0100

Refresh of move-preloadelementpcb-and

---

diff --git a/src/parse_y.y b/src/parse_y.y
index 4765ccf..0bbbfaf 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -215,32 +215,26 @@ parsepcb
 			}
 			   
 		| {
-
 		    if (yyPCB != NULL)
 		      {
-                        printf ("Begin parsing element...");
+			/* This is the case where we load a footprint with file->open, or from the command line */
 			yyFont = &yyPCB->Font;
 			yyData = yyPCB->Data;
 			yyData->pcb = yyPCB;
 			yyData->LayerN = 0;
 			layer_group_string = NULL;
 		      }
-                    else
-                      printf ("yyPCB=NULL Begin parsing element...");
 		  }
 		  element
 		  {
 		    PCBType *pcb_save = PCB;
 		    ElementType *e;
-
 		    if (yyPCB != NULL)
 		      {
-                        printf ("done\n");
-
+			/* This is the case where we load a footprint with file->open, or from the command line */
 			LayerFlag[0] = true;
 			LayerFlag[1] = true;
 			yyData->LayerN = 2;
-
 			CreateNewPCBPost (yyPCB, 0);
 			ParseGroupString("1,c:2,s", &yyPCB->LayerGroups, yyData->LayerN);
 			e = yyPCB->Data->Element->data; /* we know there's only one */
@@ -251,8 +245,6 @@ parsepcb
 			yyPCB->MaxHeight = e->BoundingBox.Y2;
 			yyPCB->is_footprint = 1;
 		      }
-                    else
-                      printf ("done, BTW: yyData is %p\n", yyData);
 		  }
 		;
